test/integration/builtins/test-amp-img.js

```diff
@@ -16,150 +16,76 @@
 
 import {AmpEvents} from '#core/constants/amp-events';
 import {createCustomEvent} from '../../../src/event-helper';
-import {
-  createFixtureIframe,
-  expectBodyToBecomeVisible,
-  poll,
-} from '#testing/iframe';
+import {createFixtureIframe, expectBodyToBecomeVisible} from '#testing/iframe';
 
-describes.sandboxed
-  .configure()
-  .enableIe()
-  .run('Rendering of amp-img', {}, () => {
-    const timeout = window.ampTestRuntimeConfig.mochaTimeout;
+describes.sandboxed.configure().run('Rendering of amp-img', {}, () => {
```

Comment:
Another instance here of the item Raghu pointed out. `describes.sandboxed.run(`

Reply:
👍

`.run()` only needs to be used after a `configure()` phrase, e.g. `configure().skipChrome().run()`. In the absence of any custom configuration, this can be simplified to just `describes.sandboxed(...)`.

Reply:
Could we add a lint check for `/\.configure\(\)h[\n ]*\.run\(/` to help folks avoid this?